### ISSUE TITLE
Video vision transformer  overfitting 

### ISSUE THREAD

**Issue by Badal-Shrestha** [keras-team/keras-io#997]
I am training hybrid vision transformer model with Dense121 model in custom dataset which contains 25 classes. The model get overfit from the start, the validation loss remains constant doesn't changes.

Each class has 60 video of length 3-5 second. 
I am following this article for training 
https://keras.io/examples/vision/video_transformers/

Is there any way to solve this issues ??

**Comment by pcoet** [comment]
Hi @Badal-Shrestha, thanks for reaching out. The issue backlog is not the best place for support requests. Can you repost this to one of the forums below? That way your question will have more visibility.

* [TensorFlow forum](https://discuss.tensorflow.org/)
* [Keras mailing list](https://groups.google.com/forum/#!forum/keras-users)

Thanks!
